src/main/resources/static/css/home-page-style.css

```
@@ -0,0 +1,81 @@
+body{
```

Prefer space between selector and brace
```suggestion
body {
```